### ISSUE TITLE
Import button is not immediately visible

### ISSUE THREAD

**Issue by CommandMC** [Heroic-Games-Launcher/HeroicGamesLauncher#1116]
### Problem description

Right now, to import a game, you first press "Install", *then* the Import button appears. This is counter-intuitive, as when you want to import a game, you don't want to click "Install".

### Feature description

Before, we had an extra button next to Install to import the game. I think we should bring back this button.

**Comment by dawidgarus** [comment]
I also find importing very confusing. When I tried import game for the first time I thought I have to set Install Path to existing folder where the existing game files are. But then when I clicked Import it prompted me to select the folder again.

But I think that having one install button is the right way. Importing is not really something that exist in other launchers and splitting it may also be confusing.

IMHO there should be a guide when you click Install button:
Step 1. Select if you wan't to a) Download the game or b) Import existing files, then click Continue.
Step 2. Selected install path and wine prefix (that's common for both a and b) and other options specific to download like language, dlcs, etc. Then click Install to either download the game or import it.

**Comment by arielj** [comment]
Maybe the action can have the text `Install/Import` in the game page, there's enough room to make it clear.

Maybe in the game card we could use a `+` icon instead of an arrow pointing down?

**Comment by flavioislima** [comment]
@biliesilva we need to figure it out how can we make importing already installed games more easier to the users.
can you think on something?

**Comment by Flexicon** [comment]
@arielj @flavioislima I think this could be closed, since the game import process is totally different now?

**Comment by arielj** [comment]
> @arielj @flavioislima I think this could be closed, since the game import process is totally different now?

This refers to the `Import Game` button that shows up in the `Install` dialog, I imagine you refer to the `Add Game` button which is not the same.

The `Import Game` button is still not clear and could be improved.

**Comment by Flexicon** [comment]
Ah I see my bad - you're right, I was referring to the `Add Game` button 👌🏻 makes sense.

Has this been looked at from design? I'd be happy to implement something here, but I wouldn't want to just roll something on my own if there is already some ideas going on around.

**Comment by arielj** [comment]
I don't think there's anything defined, I don't know what's a good UX for something like this.

What we need to solve is:
- users should be able to understand clearly that they can either download+install a game but they can also import a game that was already installed before with the game files available on disk

Currently it's not only hard to understand what to do with that `Import Game` button, it's also unclear what's going to happen and what will the user have to do

Maybe instead of having the `import game` button that takes the user out of the installation flow, we could have something next to the `Select Install Path` with a message about picking the path of an already downloaded game? then the main action of the dialog can change its text to be `Download and Install` to `Import Game` depending on what's selected as installation path?

I really don't know what's best, it feels like we have to try different ideas maybe, I don't know

**Comment by Flexicon** [comment]
Makes sense. I'll try and give it a bash, draft a PR and we can see if it makes sense in review. 🙂
